members/CTL000268.yaml

```diff
@@ -3,7 +3,7 @@ contact_form:
   method: post
   action: ""
   steps:
-    - visit: "http://www.housedems.ct.gov/Luxenberg/Contact"
+    - visit: "http://www.housedems.ct.gov/Juleson-Scopino/Contact"
```

Comment:
did a person change office?

Reply:
No, this is the same target. She just changed names, apparently.  See https://ballotpedia.org/Kelly_Juleson-Scopino